bpf/cpu-profiler/src/main.rs

```diff
@@ -32,16 +32,17 @@ pub static mut STACK_TRACES: StackTrace = StackTrace::with_max_entries(MAX_STACK
 
 #[perf_event]
 pub fn profile_cpu(ctx: PerfEventContext) -> u32 {
-    match unsafe { try_profile_cpu(ctx) } {
-        Ok(ret) => ret,
-        Err(ret) => ret,
+    unsafe {
+        try_profile_cpu(ctx);
     }
+
+    0
```

Comment:
Returning 0 all the time as the other values don't have a lot of meaning here and we can't gather them from userspace anyways.

I believe that returning non-zero might result in the kernel dropping other BPF programs waiting to execute (think of tracepoints, for example)